src/test/scala/com.socrata.geoexport/TestBase.scala

```diff
@@ -98,8 +98,8 @@ trait TestBase
       out.close()
       file
     }.toList.filter{_.getName.endsWith(".shp")}.map { f =>
-
       val store = new ShapefileDataStore(new URL("File", "", f.getAbsolutePath))
+      store.setTryCPGFile(true)
```

Comment:
Q: this call to `setTryCPGFile` is new -- can you explain why we need it?

Reply:
If you don't, it doesn't look at the CPG file and so uses the standard shapefile encoding (latin-1 rather than UTF-8)